src/DDTrace/Types.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace DDTrace\Types;
+
+const HTTP_CLIENT = 'http';
+
+const WEB_SERVLET = 'web';
+
+const SQL = 'sql';
```

Comment:
What do you mean "obfuscated"? As in, it'll be removed from the trace?

Reply:
Replacing parameters with ? and similar - see https://github.com/DataDog/datadog-trace-agent/blob/master/obfuscate/sql.go. So that the same query with different parameters is recognized as the same and grouped, I believe.